pkg/await/await.go

```diff
@@ -336,6 +336,16 @@ func Deletion(
 		// 1.6.x option. Background delete propagation is broken in k8s v1.6.
 		fg := metav1.DeletePropagationForeground
 		deleteOpts.PropagationPolicy = &fg
+	} else {
+		// > 1.7.x. Prior to 1.9.x, the default is to orphan children[1]. Our kubespy experiments
+		// with 1.9.11 is that the controller will actually _still_ mark these resources with the
+		// `orphan` finalizer, although it appears to actuually do background delete correctly. We
+		// therefore set it to background manually, just to be safe.
+		//
+		// nolint
+		// [1] https://kubernetes.io/docs/concepts/workloads/controllers/garbage-collection/#setting-the-cascading-deletion-policy
+		fg := metav1.DeletePropagationBackground
```

Comment:
`fg` doesn't make sense as a variable name here. Should probably be `bg` instead.